packages/libs/ui-kit/README.md

```diff
@@ -54,9 +55,25 @@ yarn add @ovh-ux/ui-kit
 ```js
 angular.module('myAwesomeApp', ['oui']);
 ```
+### Import only components required
+1. In your `index.html`, you need to load `oui.css` and `<component-name>.js`:
+
+```html
+<link rel="stylesheet" href="path/to/@ovh-ux/ui-kit/dist/css/oui.css">
```

Comment:
So this means that we need to bring the whole style for only one component ? Shouldn't the component bring its own style ?